bazel/external/msgpack.BUILD

```diff
@@ -0,0 +1,16 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Note to @envoyproxy/dependency-shepherds that this is a "boomerang" dependency, so we can skip the usual process for scorecard unless there is an objection.